ataru_setup.rb

```diff
@@ -0,0 +1,16 @@
+# "Require your project source code, with the correct path"
+
+require 'reek'
+
+# Ataru set up
+module Setup
+  def setup
+    # Do some nice setup that is run before every snippet
+    # If you'd like to use instance variables define them here, e.g
+    #  @important_variable_i_will_use_in_my_code_snippets = true
+  end
+
+  def teardown
+    # Do some cleanup that is run after every snippet
+  end
+end
```

Comment:
Do we need this file (and, if so, these contents)? It seems to be logically quite empty.